packages/syft/src/syft/core/node/domain_client.py

```diff
@@ -487,7 +488,10 @@ def apply_to_network(
                 raise Exception(f"No host ip in {vpn_status}")
```

Comment:
MAYBE:  Line 479, could we use the same vpn_status variable( in L462) to retrieve host ip , as self.vpn_status makes a new network call ,

Reply:
Yes, good catch. 👍

Reply:
Fixed in this commit: https://github.com/OpenMined/PySyft/pull/6567/commits/7e031150a03d995c3824c66527465d573c896035